packages/arb-token-bridge-ui/src/util/TokenTransferDisabledUtils.ts

```diff
@@ -18,15 +20,33 @@ const transferDisabledTokens: { [chainId: number]: TransferDisabledToken[] } = {
       symbol: 'FU',
       l1Address: '0x43df01681966d5339702e96ef039e481b9da20c1',
       l2Address: '0x9aee3C99934C88832399D6C6E08ad802112eBEab'
-    },
```

Comment:
Removing this since current `isTransferDisabledToken` function only takes child-chain into account. Whereas we need both child and parent chain relationship for disabling USDT deposit and withdrawal for Eth to ArbOne.

Reply:
problem with removing this is that `isTransferDisabledToken(` doesn't return `true` for USDT at any time

so the dialogs that depend on this function will not pop up

e.g. at `TokenSearch` and `TokenImportDialog`

Reply:
^ sorry i had an old version locally, it's fine